src/full-screen/components/runtime.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import screenfull from "screenfull";
+import { IAuthoredState } from "./types";
+import { FullScreenButton } from "./full-screen-button";
+import css from "./runtime.scss";
+
+interface IProps {
+  authoredState: IAuthoredState;
+}
+
+export const Runtime: React.FC<IProps> = ({ authoredState }) => {
+  const [ isFullScreen, setIsFullScreen ] = useState<boolean>(false);
```

Comment:
Do we need to maintain our own state here? It seems like it could get out of sync with the actual fullscreen state given that they're being toggled independently. I'm pretty sure the fullscreen library can be queried for the current fullscreen state. If the purpose is to trigger a render when the fullscreen status changes, you can use something like this hook:
```typescript
import { useCallback, useState } from "react";

export const useForceUpdate = () => {
  const [ , setChangeCount] = useState(0);
  return useCallback(() => setChangeCount(count => count + 1), []);
};
```